docker/README.md

```diff
@@ -0,0 +1,75 @@
+# docker-pinot
+This is a docker image of [Apache Pinot](https://github.com/apache/incubator-pinot).
+
+## How to build it
+
+There is a docker build script which will build a given Git repo/branch and tag the image.
+
+Usage:
+
+```SHELL
+./docker-build.sh [Docker Tag] [Git Branch] [Pinot Git URL]
+```
+
+This script will check out Pinot Repo `[Pinot Git URL]` on branch `[Git Branch]` and build the docker image for that.
+
+The docker image is tagged as `[Docker Tag]`.
+
+`Docker Tag`: Name and tag your docker image. Default is `pinot:latest`.
+
+`Git Branch`: The Pinot branch to build. Default is `master`.
+
+`Pinot Git URL`: The Pinot Git Repo to build, users can set it to their own fork. Please note that, the URL is `https://` based, not `git://`. Default is the Apache Repo: `https://github.com/apache/incubator-pinot.git`.
+
+Example of build and tag a snapshot on your own fork:
+```SHELL
+./docker-build.sh pinot_fork:snapshot-5.2 snapshot-5.2 https://github.com/your_own_fork/pinot.git
+```
+
+Example of publish a release version:
+```SHELL
+./docker-build.sh pinot:release-0.1.0 release-0.1.0 https://github.com/apache/incubator-pinot.git
+```
+
+There is also a `docker-build-and-push.sh` script to publish this docker image to your docker registry after build.
+
+```SHELL
+./docker-build-and-push.sh pinot:release-0.1.0 release-0.1.0 https://github.com/apache/incubator-pinot.git
+```
+
+## How to Run it
+
+The entry point of docker image is `pinot-admin.sh` script.
+
+### Bring up Zookeeper
+Example of bring up a local zookeeper in docker:
+```SHELL
+docker pull zookeeper
+docker run --name  pinot-zookeeper --restart always -p 2181:2181  zookeeper
+```
+You can extract the zookeeper host from:
+```SHELL
+docker inspect pinot-zookeeper|grep IPAddress
+            "SecondaryIPAddresses": null,
+            "IPAddress": "172.17.0.2",
+                    "IPAddress": "172.17.0.2",
```

Comment:
Duplicate line? Also, extra `,`?

Reply:
Not really, this field appears also inside the nested field:
Below is a sample  example of `NetworkSettings`
```
         "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "23406af2ceaa015751faab22e383216e173c111073510c0134aa8d8189acce47",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {
                "2181/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "2191"
                    }
                ],
                "2888/tcp": null,
                "3888/tcp": null
            },
            "SandboxKey": "/var/run/docker/netns/23406af2ceaa",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "72e7e3628415fcf12c972ac5e8ceaa5536cfdb6d90df6f3b13f5f05bcdd36628",
            "Gateway": "172.17.0.1",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "172.17.0.3",
            "IPPrefixLen": 16,
            "IPv6Gateway": "",
            "MacAddress": "02:42:ac:11:00:03",
            "Networks": {
                "bridge": {
                    "IPAMConfig": null,
                    "Links": null,
                    "Aliases": null,
                    "NetworkID": "b6b2f7c3d4c82fd85370d998f0de3d6a517f0350c0495169a1d7751968ee24bb",
                    "EndpointID": "72e7e3628415fcf12c972ac5e8ceaa5536cfdb6d90df6f3b13f5f05bcdd36628",
                    "Gateway": "172.17.0.1",
                    "IPAddress": "172.17.0.3",
                    "IPPrefixLen": 16,
                    "IPv6Gateway": "",
                    "GlobalIPv6Address": "",
                    "GlobalIPv6PrefixLen": 0,
                    "MacAddress": "02:42:ac:11:00:03",
                    "DriverOpts": null
                }
            }
        }
```